python/yugabyte_db_thirdparty/library_checking.py

```diff
@@ -93,8 +93,10 @@ def compile_re_list(re_list: List[str]) -> Any:
 
 
 def get_needed_libs(file_path: str) -> List[str]:
+    env = {"PATH": os.getenv("PATH")}
     return capture_all_output(
         ['patchelf', '--print-needed', file_path],
+        env=env,
```

Comment:
Why do we need to remove all environment variables except PATH?

Reply:
We don't per se, but in my use case the PATH variable was not being picked up unless I explicitly included it, like so